src/classes/TDTM_DefaultConfig.cls

```diff
@@ -37,6 +37,36 @@
                settings and the new ones.
 */
 public without sharing class TDTM_DefaultConfig {
+    
+    public static boolean ADDR_Address_TDTM_TestAsync = false;
+    private static boolean ADDR_Address_TDTM_TestAsyncTemp;
```

Comment:
This might just be me being slow this morning but, don't static boolean variables get initialized to false if no value is given? And if so, is there really any way that these temp variables could ever be set to true given this logic?